lib/frozen_record/base.rb

```diff
@@ -1,5 +1,6 @@
 require 'set'
 require 'active_support/descendants_tracker'
+require 'frozen_record/backends/json'
```

Comment:
I think we should use autoload for the backends. Not all apps will use both backends.

Reply:
Done :)